addons/mail/static/src/model/make_store.js

```diff
@@ -62,6 +54,9 @@ export function makeStore(env, { localRegistry } = {}) {
                          */
                         get(record, name, recordFullProxy) {
                             recordFullProxy = record._.downgradeProxy(record, recordFullProxy);
+                            if (record._.gettingField || !Model._.fields.get(name)) {
```

Comment:
Shouldn't it be scoped per field?

Reply:
It's just a reflect get for non-field value on record. This is synchronous and immediate without any side-effect, so it's overkill to scope it per field

Reply:
Why not use the reflect directly rather than setting the flag and entering the proxy again then?